edit-post/store/test/reducer.js

```diff
@@ -153,6 +154,23 @@ describe( 'state', () => {
 		} );
 	} );
 
+	it( 'should set the activeModal to the provided name', () => {
```

Comment:
Can you please group these tests into a `describe( 'activeModal', () => { ... } )`?